automatic/1password/1password.nuspec

```diff
@@ -3,10 +3,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>1password</id>
-    <version>7.9.832</version>
+    <version>8.8.0</version>
     <title>1Password</title>
-    <authors>AgileBits</authors>
-    <owners>chocolatey-community, AgileBits</owners>
+    <authors>1Password</authors>
```

Comment:
Since you are updating the `authors`, remember to update the `copyright` as well.

Looking at the website the copyright should be:

```
© 2022 1Password. All rights reserved.
```